index.html

```diff
@@ -37,9 +43,235 @@ <h1>Donice Ubaru</h1>
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+
+      <div>
+        <img src="./images/project-image-3.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+
+      <div>
+        <img src="./images/project-image-4.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+
+      <div>
+        <img src="./images/project-image-5.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
+          </button>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section>
+
+    <h2>About</h2>
+    <div id="About-myself">
+      <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <button type="button">
+        Get my CV
+        <span class="material-symbols-outlined">download</span>
+      </button>
+
+      <div class="skill">
+        <div class="skill-card" id="language">
+          <h4>Languages</h4>
+          <ul>
+            <li>Javascript(ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+
+        <div class="skill-card" id="frameworks">
+          <h4>Frameworks</h4>
+          <ul>
+            <li>React.js</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+          </ul>
+        </div>
+
+        <div class="skill-card" id="skills">
```

Comment:
As requested by the previous reviewer, please kindly increase the `font-weight` for categories in the "About myself" section.

Your design
![image](https://github.com/donice/Portfolio-Microverse/assets/78251927/7937ea01-f66d-4d3d-b0f9-b52a5ec7c26e)

Figma
![image](https://github.com/donice/Portfolio-Microverse/assets/78251927/eb46e095-77ae-4241-825a-4b3655777af6)